BUILDING.md

```diff
@@ -36,18 +36,23 @@ build.cmd -help
 1. Rust\
 Install Rust using [rustup](https://rustup.rs/).
 
-2. .NET SDK 8\
+2. .NET SDK 8.0\
 Download the .NET SDK from [dotnet.microsoft.com](https://dotnet.microsoft.com/en-us/download/dotnet).
 
-3. docfx (optional)\
-The IceRpc API reference is generated by [docfx](https://www.nuget.org/packages/docfx). You can install docfx as
+3. Rosetta 2 on Apple silicon Mac computers\
+Make sure you install [Rosetta 2] before building IceRPC on macOS for Apple silicon. It is required for running protoc
+when building the IceRPC + Protobuf integration. Corresponding protoc issue:
```

Comment:
> It is required for running protoc when building the IceRPC + Protobuf integration.

Is this true, I thought it was only require for the examples that use Protobuf files, but we don't need to run protoc for building the integration package

Reply:
You're right, it's incorrect. We need it when building the tests.